users/templates/accounts/success-registration.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load rest_framework %}
```

Comment:
Why you loading here `rest_framework`? I think that it unnecessary here.

Reply:
Removed